packages/builders/src/messages/embed/UnsafeEmbed.ts

```diff
@@ -299,7 +300,7 @@ export class UnsafeEmbed implements Equatable<APIEmbed | UnsafeEmbed> {
 	 *
 	 * @param timestamp The timestamp or date
 	 */
-	public setTimestamp(timestamp: number | Date | null = Date.now()): this {
+	public setTimestamp(timestamp: number | Readonly<Date> | null = Date.now()): this {
```

Comment:
We really need [ReadonlyDate](https://github.com/microsoft/TypeScript/issues/29274), heh.

`Readonly<Date>` is a bit useless here, it doesn't have public fields, only methods.